preprocessors/graphic-tagger.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/graphic-tagger.schema.json",
+    "type": "object",
+    "title": "Graphic Tagger",
+    "description": "Preprocessor for obtaining tags for the content in the image",
```

Comment:
This is very vague. You should also probably address why "people" is mutually exclusive with "indoor" and "outdoor". This partly serves as documentation.

Reply:
This is a rough schema and will be changed in the future after we decide how to use tags. I don't think further justification for labels is needed at this point.